includes/wikia/models/DesignSystemGlobalFooterModelV2.class.php

```diff
@@ -533,6 +545,18 @@ private function getCommunity() {
 			];
 		}
 
+		if ( true ) {
+			$data['links'][] = [
```

Comment:
why are you adding this link twice?

Reply:
Once for *fandom.com and once for *.wikia.org
It's not decided yet if we want to display it on *.wikia.org. My guess is "No". When the decision is made I will remove/leave it.

Reply:
Link will appear on both fandom.com and wikia.org.